addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss

```diff
@@ -266,33 +265,38 @@
 }
 @media only screen and (max-width: 767px) {
     .oe-toolbar {
-        position: relative;
+        overflow-x: auto;
         visibility: visible;
         width: 100%;
+        border-bottom: 1px solid $border-color;
         border-radius: 0;
         background-color: white;
-
+        box-shadow: none;
+        height: auto;
+        
         .btn {
             color: black;
+            padding: 3.5px 4px !important;
+        }
+        .dropdown-menu{
+            position: fixed !important;
         }
     }
 }
-
+@media only screen and (min-width: 767px) {
+    .oe-toolbar.oe-floating{
+        box-shadow: rgba(0, 0, 0 , 0.1) 0px 4px 12px, rgba(0, 0, 0, 0.05) 0px 0px 4px;
+    }
+}
 /* Content styling */
 
 .oe-powerbox-wrapper {
-    position: absolute;
     z-index: $zindex-modal;
-    border: black;
     background: var(--oePowerbox__wrapper-bg, $o-white);
     color: $o-main-text-color;
     max-height: 40vh;
     box-sizing: border-box;
-    max-width: 100%;
-    box-shadow: 0px 3px 18px rgba(0, 0, 0, .23);
-    border-radius: 4px;
-    overflow: hidden;
-    display: flex;
+    box-shadow: rgba(0, 0, 0 , 0.1) 0px 4px 12px, rgba(0, 0, 0, 0.05) 0px 0px 4px;
```

Comment:
can't we use a combination of these variables? 
https://github.com/twbs/bootstrap/blob/1a6fdfae6b/scss/_variables.scss#L538

Reply:
I replaced it by the $box-shadow variable but I think we should change that variable to make it less harsh.